docs/source/markdown/podman-diff.1.md

```diff
@@ -46,8 +49,13 @@ A /usr/local/bin/docker-entrypoint.sh
 }
 ```
 
+```
+$ podman diff container1 image1
+A /test
+```
+
 ## SEE ALSO
-podman(1)
+podman(1), podman-container(1), podman-image(1)
```

Comment:
Did you want container or container-diff here?  I went with container-diff, but maybe both?
```suggestion
**[podman(1)](podman.1.md)**, **[podman-container-diff(1)](podman-container-diff.1.md)**, **[podman-image(1)](podman-image.1.md)**
```